Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -41,9 +41,12 @@
 //
 // Limit Switches
 //
-#define X_STOP_PIN         35
-#define Y_STOP_PIN         12
-#define Z_STOP_PIN         36
+#define X_MIN_PIN          35
+#define X_MAX_PIN          -1
+#define Y_MIN_PIN          -1
+#define Y_MAX_PIN          12
+#define Z_MIN_PIN          36
+#define Z_MAX_PIN          -1
```

Comment:
Let's try to make it work as intended rather than sacrifice the option for Printrboard. So the question is, how is the code in `pins.h` (pasted below) failing to set the correct pin number for `Y_MAX` or `Z_MIN`? Do we perhaps need to account for `USE_YMAX_PLUG` and `USE_ZMIN_PLUG` being set as well?